community/blog/2023-01 Role-based authentication in Azure API Management made easy in Arcus Scripting v0.8/README.md

```diff
@@ -0,0 +1,63 @@
+# Role-based authentication in Azure API Management made easy in Arcus Scripting v0.8
+Managing roles during JWT validation was tedious and repetitive work. Luckily, the newest Arcus Scripting release changed that.
+
+## Validating JSON web tokens in Azure API Management
+The `validate-jwt` policy in Azure API Management provides the capability to consumers to enforce a valid JWT in an incoming HTTP request. One of these validation rules is whether certain roles have been assigned to a service principal. This authorization functionality is very useful to allow/deny certain functionality to users of the application.
+
+Management of service principal roles to certain Azure Directory Applications is rather tedious. One has to look up the role assignments of an Azure Active Directory application to find out if the service principal has the correct access. Moreover, in certain scenarios, one has to wait a couple of seconds before a role assignment is available for use. All this adds to the problem of managing a service principal for validating JWTs in Azure API Management.
+
+An example of such policy is shown here:
+
+```xml
```

Comment:
I would move this part below `The validate-jwt policy in Azure API Management provides the capability to consumers to enforce a valid JWT in an incoming HTTP request. One of these validation rules is whether certain roles have been assigned to a service principal. This authorization functionality is very useful to allow/deny certain functionality to users of the application.` part

Reply:
Moved `validate-jwt` example below 1st paragraph in https://github.com/arcus-azure/arcus/pull/290/commits/36768f2ba60ffd9670f8d6e3bc89247c3a7aa6e7